### ISSUE TITLE
Office.EventType.ItemChanged loose focus after clicked href

### ISSUE THREAD

**Issue by oliverbj** [OfficeDev/office-js#3002]
# Provide required information needed to triage your issue
When a button inside the taskpane of my Outlook add-in is clicked, the `Office.EventType.ItemChanged` is no longer firing when the selection changes to a new email.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]:  PC desktio
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Version 2209 /Build 15629.20258)
* Operating System: Windows 10 Enterprise
* Browser (if using Office on the web): N/A
 
## Expected behavior

I would expect that no matter what I click on in the Outlook add-in, that the events are still firing and working.

## Current behavior

I am using Office.js to develop a custom Outlook Add-In. A part of my script includes an event handler, to detect whenever a new e-mail has changed:

```javascript
Office.context.mailbox.addHandlerAsync(Office.EventType.ItemChanged, selectedMailItemChanged);

async function selectedMailItemChanged(eventArgs){
    console.log("E-mail was selected for the first time, or the email selection has changed.");
}
```

The above code works fine. When I select a new e-mail, the `selectedMailItemChanged` function gets triggered, and writes to the console.

However, it stops working on specific instances, such as when I click on a link. In one of my pages, I have the below link:

```html
<a
:href="'edient:Command=ShowEditForm&LicenceCode=XXXXXX&ControllerID=JobShipment&BusinessEntityPK=' + shipment.fetch_id">
 <span>Open Shipment</span>
</a>
```

The above href links to an internal application. The link works fine, but after it is clicked, the `selectedMailItemChanged` function no longer fires when the selection changes.

## Steps to reproduce

1. ______
2. ______
3. ______

## Link to live example(s)

1. ______
2. ______
3. ______

# Provide additional details

1. ______
2. ______
3. ______

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by davidchesnut** [comment]
Hi @oliverbj, thanks for letting us know about this issue. @exextoc can you please help resolve this? Thanks!

**Comment by exextoc** [comment]
@oliverbj Can you share a repro video and exact steps?

**Comment by oliverbj** [comment]
@exextoc please see below

https://user-images.githubusercontent.com/7851696/204554693-00b64fa0-813f-4136-bad4-0722b484c02d.mp4

**Comment by exextoc** [comment]
Thanks for the video. We are able to reproduce this issue in-house. It has been placed on our backlog. We unfortunately have no timelines to share at this point.

Internal tracking id: Office: 7260923

**Comment by exextoc** [comment]
@oliverbj this issue has been fixed in build 16.0.16113.10000.
